code/modules/mod/modules/modules_antag.dm

```diff
@@ -545,3 +545,54 @@
 
 /obj/item/gun/medbeam/mod
 	name = "MOD medbeam"
+
+/obj/item/mod/module/stealth/wraith
+
+	desc = "A more destructive adaptation of the stealth module. The Wraith Module does not simply bend light around the user to obscure their visual pattern, but actively attacks and overloads surrounding light emitting objects, repurposing this energy to power the suit. It is possible that this technology has its origins in Spider Clan advancements, but the exact source of the Wraith Module is highly disputed. No group has stepped forward to claim it as their handiwork due to the political consequences of having stolen Spider Clan tech and their inevitable retaliation for such transgressions. Most point fingers at Cybersun Industries, but murmurs suggest it could even be even more clandestine organizations amongst the Syndicate branches. Whatever the case, if you are looking at one of these right now, don’t show it to a space ninja."
```

Comment:
Geeze, this description is even longer than most emotes from "extended" roleplay on furry servers. Can you move it from `examine()` to `examine_more()`?